packages/utils/README.md

```diff
@@ -0,0 +1,37 @@
+<h1 align="center">Utils for ESLint Plugins</h1>
+
+<p align="center">Utilities for working with TypeScript + ESLint together.</p>
+
+<p align="center">
+    <img src="https://github.com/typescript-eslint/typescript-eslint/workflows/CI/badge.svg" alt="CI" />
+    <a href="https://www.npmjs.com/package/@typescript-eslint/utils"><img src="https://img.shields.io/npm/v/@typescript-eslint/utils.svg?style=flat-square" alt="NPM Version" /></a>
+    <a href="https://www.npmjs.com/package/@typescript-eslint/utils"><img src="https://img.shields.io/npm/dm/@typescript-eslint/utils.svg?style=flat-square" alt="NPM Downloads" /></a>
+</p>
+
+## Note
+
+This package has inherited its version number from the `@typescript-eslint` project.
+Meaning that even though this package is `5.x.y`, you shouldn't expect 100% stability between minor version bumps.
```

Comment:
I think the whole point of this change is that this is no longer correct.

If we believe this statement to be true, we should _not_ change the name yet.

If we are ready to make the name change, we should delete all of this breaking change notice as `utils` would follow the same major version conventions as the other packages.

Reply:
cc @bradzacher

Reply:
yuppo that's correct.
TBF we've treated this package as completely stable since like v2.0 haha
This should defs be removed